qcodes/station.py

```diff
@@ -1,17 +1,45 @@
 """Station objects - collect all the equipment you use to do an experiment."""
-from typing import Dict, List, Optional, Sequence, Any
-
+from contextlib import suppress
+from typing import Dict, List, Optional, Sequence, Any, cast, AnyStr, IO
+from functools import partial
+import importlib
+import logging
+import os
+from copy import deepcopy
+from typing import Union
+
+import qcodes
 from qcodes.utils.metadata import Metadatable
-from qcodes.utils.helpers import make_unique, DelegateAttributes
+from qcodes.utils.helpers import make_unique, DelegateAttributes, YAML
 
 from qcodes.instrument.base import Instrument
-from qcodes.instrument.parameter import Parameter
-from qcodes.instrument.parameter import ManualParameter
-from qcodes.instrument.parameter import StandardParameter
+from qcodes.instrument.parameter import (
+    Parameter, ManualParameter, StandardParameter,
+    DelegateParameter)
+import qcodes.utils.validators as validators
+from qcodes.monitor.monitor import Monitor
 
 from qcodes.actions import _actions_snapshot
 
 
+log = logging.getLogger(__name__)
+
+PARAMETER_ATTRIBUTES = ['label', 'unit', 'scale', 'inter_delay', 'delay',
```

Comment:
There's no `delay` in parameter, but there is a post delay.
```suggestion
PARAMETER_ATTRIBUTES = ['label', 'unit', 'scale', 'inter_delay', 'post_delay',
```

Reply:
Thanks for catching that. There used to be delay at the time of writing....